src/TransactionsSummary/AmountSummary/AmountSummary.jsx

```diff
@@ -0,0 +1,14 @@
+import classes from './styles.module.css';
+export function AmountSummary({ transactions }) {
+	const total = transactions.reduce(
```

Comment:
Bardzo fajnie do tego podszedles. 
To co bym zasugerowal to ten kawalek kodu `transactions.reduce(...` wrzucilbym do funkcji w stylu `calculateTransactionTotalAmount` dzieki temu kod ktory piszesz bedzie czytelniejszy.

Reply:
Bedzie po prostu lepiej informowal jaka byla Twoja intencja